config.py

```diff
@@ -34,7 +34,6 @@
         % POSTGRES
     )
 
-SQLALCHEMY_TRACK_MODIFICATIONS = True
```

Comment:
Removed the `SQLALCHEMY_TRACK_MODIFICATIONS` because getting an error for the use of this after upgrading `flask-SQLalchemy`.